docs/source/quickstart.rst

```diff
@@ -3,6 +3,219 @@
 Quickstart
 ==========
 
+Install MLFlow, instrument your code, and view a run's results in a few minutes:
+
+
+How do I install MLFlow? 
+------------------------
+
+You may install MLflow from PyPi using pip:
```

Comment:
We probably want to mention that there's an R and Java library, but we don't need to show the examples in R or Java.

Reply:
Done in intro paragraphs and drill down.